editor/plugins/script_editor_plugin.cpp

```diff
@@ -747,11 +747,13 @@ void ScriptEditor::_close_tab(int p_idx) {
 		history_pos = history.size() - 1;
 	}
 
-	int idx = tab_container->get_current_tab();
+	tab_container->set_current_tab(0);
 	memdelete(tselected);
+	int idx = tab_container->get_current_tab();
```

Comment:
it seems so to me.
I don't know how hard to do it, but anyway, it would be better to select previous selected script if any.